core/controllers/admin.py

```diff
@@ -854,3 +854,23 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to retrive the list of interaction id's for a exploration id."""
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id', None)
+        if exploration_id is None:
```

Comment:
Are we expecting this to be checking whether exploration_id is not a string, right?

Reply:
yes i'll add one more for checking string